CONTRIBUTING.md

```diff
@@ -1,195 +1,116 @@
-
-**Prerequisite 2** - Node (v8.9.x or latest LTS release) + npm (v5.6.0 or later)
-
-**Prerequisite 3** - Docker 
-
-#### Building from the source
-
-Clone this repository using the following command.
-
-```
-git clone --recursive https://github.com/ballerina-platform/ballerina-lang.git
-```
-
-If you have forked the repository to your github account then use the following command replacing <YOUR-GITHUB-USERNAME> with your git username.
-
-````
-git clone --recursive https://github.com/<YOUR-GITHUB-USERNAME>/ballerina-lang.git
-````
-
-If you download the sources, you need to update the git submodules using the following command.
-
-```
-git submodule update --init 
-```
-
-Run the gradle command `./gradlew clean build` from the Ballerina root directory:
-
-Extract the Ballerina distribution created at `distribution/zip/ballerina/target/ballerina-<version>-SNAPSHOT.zip`. The `zip/ballerina` directory contains the runtime only. `zip/ballerina-tools/` contains the runtime and tools (e.g., Ballerina Composer).
-
-Note: It is possible to face an IOException error stating "Too many open files". This is due to the default number of possible open files being set to a lower number on your operating system than required for Ballerina to be compiled. You may have to increase the number of open files/file descriptors (FD) on your operating system. This is OS dependent and you can search the internet on how to do this for your operating system (Windows/OSX/Linux). You could update the maximum number of open files to 1000000 (or higher).
-
-### Setting up your development environment
-
-#### Setup IntelliJ IDEA
-
-##### Importing a Ballerina Project
+- [Get started](#get-started)
+- [Set up the Prerequisites](#set-up-the-prerequisites)
+- [Obtain the source code](#obtain-the-source-code)
+- [Build the project](#build-the-project)
+- [Set up the development environment](#set-up-the-development-environment)
+    - [Set up the Ballerina plugin](#set-up-the-ballerina-plugin)
+    - [Set up the ANTLR4 plugin](#set-up-the-antlr4-plugin)
+- [Submit your contribution](#submit-your-contribution)
+- [Accept the Contributor License Agreement](#accept-the-contributor-license-agreement)
+- [Propose changes to Ballerina](#propose-changes-to-ballerina)
 
-Import the Ballerina project to IntelliJ IDEA similar to any other Gradle project. 
+## Get started
 
-* Navigate to `File | Open` menu.
-* Browse the filesystem and Select Ballerina Project's root directory.
-   * If prompted to import Gradle project select `Auto import`.
-   * Set Project SDK as Java 1.8
+- Join the conversations at:
 
-##### Useful IDEA Plugins
+    - [StackOverflow](https://stackoverflow.com/questions/tagged/ballerina): to get help with Ballerina (use the Ballerina tag for any of your questions.)
+    - [Slack](https://ballerinalang.slack.com/): for real-time discussions with the team and community
```

Comment:
Incorrect url? The correct url is `https://ballerina-platform.slack.com/`. Btw, IMO we should have `https://ballerina.io/community/slack/` instead of that because then users can get the invitations by registering.

Reply:
Changed to https://ballerina.io/community/slack/.